docs-client/src/lib/transports/transport.ts

```diff
@@ -47,8 +47,37 @@ export default abstract class Transport {
     return this.doSend(method, filledHeaders, bodyJson, endpointPath, queries);
   }
 
+  public curlPathAndBody(method: Method, body: string): [string, string] {
+    const endpoint = this.findDebugMimeTypeEndpoint(method);
+    if (!endpoint) {
+      throw new Error("Endpoint null");
+    }
+
+    return [this.getCurlPath(endpoint), this.getCurlBody(body, endpoint, method)];
+  }
+
+  protected findDebugMimeTypeEndpoint(method: Method): Endpoint {
+    const endpoint = method.endpoints.find((ep) =>
+        ep.availableMimeTypes.includes(this.getDebugMimeType()),
+    );
+    if (!endpoint) {
+      throw new Error('Endpoint does not support debug transport.');
```

Comment:
How about `Endpoint does not support debug transport. MimeType: ${this.getDebugMimeType()}`?

Reply:
Also, Could you remove the duplicate code in grpc-unframed.ts, please?

Reply:
Also, removed from `annotated-http.ts`